crates/meroctl/src/cli/context.rs

```diff
@@ -38,6 +42,12 @@ pub const EXAMPLES: &str = r"
 
   # Create a new context in dev mode
   $ meroctl --  --node-name node1 context create --watch <path> -c <contextId>
+
+  # Grant permission to manage applications
+  $ meroctl context grant --granter alice --grantee bob ManageApplication
+
+  # Revoke permission to manage members
+  $ meroctl context revoke --granter alice --grantee bob ManageMembers
```

Comment:
Seeing it this way, I'm inclined to go with this:

```suggestion
  # Grant permission to manage applications
  $ meroctl context grant bob ManageApplication --as alice

  # Revoke permission to manage members
  $ meroctl context revoke bob ManageMembers --as alice
```